src/main/kotlin/fi/metatavu/edufication/pages/api/impl/V1ApiImpl.kt

```diff
@@ -0,0 +1,103 @@
+package fi.metatavu.edufication.pages.api.impl
+
+import fi.metatavu.edufication.pages.api.controller.PagesController
+import fi.metatavu.edufication.pages.api.impl.translate.ContentBlockTranslator
+import fi.metatavu.edufication.pages.api.impl.translate.PageTranslator
+import fi.metatavu.edufication.pages.api.model.Page
+import fi.metatavu.edufication.pages.api.spec.V1Api
+import org.hibernate.annotations.NotFound
+import org.jboss.resteasy.util.NoContent
+import java.util.*
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+import javax.transaction.Transactional
+import javax.ws.rs.BadRequestException
+import javax.ws.rs.core.Response
+
+@RequestScoped
+class V1ApiImpl: V1Api, AbstractApi()  {
+
+    @Inject
+    private lateinit var pagesController: PagesController
+
+    @Inject
+    private lateinit var pageTranslator: PageTranslator
+
+    @Inject
+    private lateinit var contentBlockTranslator: ContentBlockTranslator
+
+    @Transactional
+    override fun createPage(page: Page): Response {
+        val userId = loggerUserId ?: return createUnauthorized(NO_VALID_USER_MESSAGE)
+
+        val createdPage = pagesController.create(
+            status = page.status,
+            path = page.path,
+            creatorId = userId,
+            contentBlocks = page.contentBlocks
+        )
+
+        val translated = pageTranslator.translate(createdPage)
+        translated.contentBlocks = contentBlockTranslator.translate(pagesController.getPageContent(translated.id))
```

Comment:
If page always includes contentBlocks, this translation should be moved to page translator